src/vs/monaco.d.ts

```diff
@@ -5354,7 +5354,7 @@ declare namespace monaco.languages {
 	/**
 	 * Register a code action provider (used by e.g. quick fix).
 	 */
-	export function registerCodeActionProvider(languageId: string, provider: CodeActionProvider): IDisposable;
+	export function registerCodeActionProvider(languageId: string, provider: CodeActionProvider, metadata?: CodeActionProviderMetadata): IDisposable;
```

Comment:
I added these by hand, but I'm unsure if there is a generation process of this file involved somewhere. Or is it only the final monaco.d.ts that's shipped with the NPM package that's generated from this one?

Reply:
I think it is being generated, but the outcome would be the same.